components/brave_wallet_ui/page/async/wallet_page_async_handler.ts

```diff
@@ -34,7 +34,7 @@ async function refreshWalletInfo (store: Store) {
 handler.on(WalletPageActions.createWallet.getType(), async (store, payload: CreateWalletPayloadType) => {
   const apiProxy = await getAPIProxy()
   const result = await apiProxy.createWallet(payload.password)
-  store.dispatch(WalletActions.updateWalletNames({ accountNames: ['Account 1'] }))
+  store.dispatch(WalletActions.setInitialAccountNames({ accountNames: ['Account 1'] }))
```

Comment:
in addition to https://github.com/brave/brave-core/pull/9395/files#r673584352, `setInitialAccountNames` doesn't check to see if `createWallet` was successful or not (non-empty result.mnemonic)